qadence_protocols/mitigations/twirl.py

```diff
@@ -40,7 +46,7 @@ def mitigate(
     backend: BackendName,
     noise: Noise,
     n_shots: int,
-    observable: list,
+    observables: list,
```

Comment:
I think this has to change wrt the new API design. You can have a look at an example in `readout.py`. Basically, only the model, noise and few other arguments need to be passed, tobe able to do:

```python
mitigated = mitigate(model=model, noise=noise, ...)
```

Also, please have a look at `test_readout.py` for the usage.